Userland/Libraries/LibWeb/HTML/DataTransferItemList.cpp

```diff
@@ -108,4 +110,16 @@ JS::GCPtr<DataTransferItem> DataTransferItemList::add(JS::NonnullGCPtr<FileAPI::
     return item;
 }
 
+// https://html.spec.whatwg.org/multipage/dnd.html#dom-datatransferitemlist-item
+Optional<JS::Value> DataTransferItemList::item_value(size_t index) const
+{
+    // To determine the value of an indexed property i of a DataTransferItemList object, the user agent must return a
+    // DataTransferItem object representing the ith item in the drag data store. The same object must be returned each
+    // time a particular item is obtained from this DataTransferItemList object. The DataTransferItem object must be
+    // associated with the same DataTransfer object as the DataTransferItemList object when it is first created.
+    if (index <= m_data_transfer->length())
```

Comment:
`if (index < m_data_transfer->length())`?

Reply:
:man_facepalming:  wow, thanks! Fixed, and included the edge case in the test